src/components/commons/StyledButton.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const DefaultProps = {
+    background: {
+        startColor: '#2cd889',
+        endColor: '#4ca07a'
+    }
+}
+
+function StyledButton({children, onClick, stylingProps}) {
```

Comment:
StyledButton is not util, so we might move it to components package instead of commons

Reply:
This button is used in around 3 places and has same background styling. So, I have thought of moving it to common components section